zerver/lib/event_queue.py

```diff
@@ -146,21 +156,25 @@ def accepts_event(self, event):
 
     # TODO: Refactor so we don't need this function
     def accepts_messages(self):
+        # type: () -> bool
         return self.event_types is None or "message" in self.event_types
 
     def idle(self, now):
+        # type: (float) -> bool
         if not hasattr(self, 'queue_timeout'):
             self.queue_timeout = IDLE_EVENT_QUEUE_TIMEOUT_SECS
 
         return (self.current_handler_id is None
                 and now - self.last_connection_time >= self.queue_timeout)
 
     def connect_handler(self, handler_id, client_name):
+        # type: (int, str) -> None
```

Comment:
Is `client_name` of the same type as [zerver.models.Client.name](https://github.com/zulip/zulip/blob/2080038e7c5fb2fdcaeb06a9b14d29e01e6535e0/zerver/models.py#L643)? If yes, then it should be `text_type`. @timabbott maybe you know this.